components/tools/OmeroPy/src/omero/plugins/chown.py

```diff
@@ -65,10 +75,65 @@ def cmd_type(self):
         import omero.all
         return omero.cmd.Chown2
 
+    def _configure(self, parser):
+        parser.set_defaults(func=self.main_method)
+        self._add_wait(parser, default=-1)
+        parser.add_argument(
+            "--include",
+            help="Modifies the given option by including a list of objects")
+        parser.add_argument(
+            "--exclude",
+            help="Modifies the given option by excluding a list of objects")
+        parser.add_argument(
+            "--ordered", action="store_true",
+            help=("Pass multiple objects to commands strictly in the order "
+                  "given, otherwise group into as few commands as possible."))
+        parser.add_argument(
+            "--list", action="store_true",
+            help="Print a list of all available graph specs")
+        parser.add_argument(
+            "--list-details", action="store_true", help=SUPPRESS)
+        parser.add_argument(
+            "--report", action="store_true",
+            help="Print more detailed report of each action")
+        parser.add_argument(
+            "--dry-run", action="store_true",
+            help=("Do a dry run of the command, providing a "
+                  "report of what would have been done"))
+        parser.add_argument(
+            "--force", action="store_true",
+            help=("Force an action that otherwise defaults to a dry run"))
+        self._pre_objects(parser)
+        parser.add_argument(
+            "obj", nargs="*", type=GraphArg(self.cmd_type()),
+            help="objects to be processed in the form <Class>:<Id>"
+                 " and/or user(s) to transfer all data from in the"
+                 " form Experimenter:<Id>")
+
     def _pre_objects(self, parser):
         parser.add_argument(
             "usr", nargs="?", type=ExperimenterArg,
-            help="""user to transfer ownership of objects to""")
+            help="user to transfer ownership of specified objects and/or all"
+                 " objects owned by specified user(s) to")
+
+    def populate_targetUsers(self, command_check):
+        # Move the Experimenters whose data are to be
+        # transferred from targetObjects
+        # to targetUsers. The rewritten _check_command
+        # method checked these Experiments when they were
+        # in targetObjects, but Chown2 needs them in targetUsers
+        for k, v in command_check.targetObjects.items():
```

Comment:
This is the ``req`` - see below, we were wondering what type of object that (req.targetUsers) is. 
```
bin/omero chown 2 Image:1,51 Dataset:105,106 Experimenter:52,2,102 --report --dry-run
Using session fdca147e-2f7b-4a52-8294-5a28eff7d38c (root@localhost:4064). Idle timeout: 10 min. Current group: system
object #0 (::omero::cmd::Chown2)
{
    targetObjects = 
    {
        key = Image
        value = 
        {
            [0] = 1
            [1] = 51
        }
        
        key = Dataset
        value = 
        {
            [0] = 105
            [1] = 106
        }
    }
    childOptions = {}
    dryRun = True
    userId = 2
    targetUsers = 
    {
        [0] = 52
        [1] = 2
        [2] = 102
    }
}
```